### PR TITLE
fix the height of main menu in toolbar

### PR DESCRIPTION
**Pull request by fkroockmann**
Too many spaces into the menu of toolbar

### CODE REVIEW

File: less/bb-ui.less [hunk 1]
```diff
@@ -71,7 +71,7 @@ Version: 2.10 Dec2014
 .make-border-shadow {
     background: -moz-linear-gradient(top,  rgba(218,218,218,1) 0%, rgba(0,0,0,0) 100%); /* FF3.6+ */
     background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(218,218,218,1)), color-stop(100%,rgba(0,0,0,0))); /* Chrome,Safari4+ */
-    background: -webkit-linear-gradient(top,  rgba(218,218,218,1) 0%,rgba(0,0,0,0) 100%); /* Chrome10+,Safari5.1+ */
+    background: -webkit-linear-gradient(top,  rgba(218,218,218,1) 0%rgba(0,0,0,0) 100%); /* Chrome10+,Safari5.1+ */
```
Comment (ndufreche) [inline less/bb-ui.less]: this work ?

Comment (fkroockmann) [inline less/bb-ui.less]: Its a bug imo, its the same line

Comment (ndufreche) [inline less/bb-ui.less]: is not the same line, you remove the comma.

Comment (fkroockmann) [inline less/bb-ui.less]: ok, yes its work...

Comment (ndufreche) [inline less/bb-ui.less]: You don't really know :p
/* Chrome10+,Safari5.1+ */
I know about your computer system, you can't test that :)
This fix is for safari (mac / window) and chrome > 10 and < 31 and I'm pretty sure your chrome is over version 31.

Comment (fkroockmann) [inline less/bb-ui.less]: Julien know what he do :/

Comment (ndufreche) [inline less/bb-ui.less]: @jtrolle we need your verdict here

Comment (jtrolle) [inline less/bb-ui.less]: the good writing is the one with the comma

Comment (ndufreche) [inline less/bb-ui.less]: Thanks, @jtrolle

**Pull request merged**